families/supplychain/restapi/openapi.yaml

```diff
@@ -0,0 +1,398 @@
+swagger: '2.0'
+
+info:
+  version: "0.8.0"
+  title: Sawtooth Supplychain REST API
+  description: |
+    _This HTTP pragmatic REST API is built on top of Sawtooth Lake's existing
+    ZMQ/Protobuf infrastructure, simplifying SupplyChain client interactions
```

Comment:
Should this be against the Sawtooth REST API?  I'm not sure we are promoting client applications directly against the ZMQ interface, just yet (there's no easy SDK method for this).